src/bundler/bundle_v2.zig

```diff
@@ -2501,6 +2526,116 @@ pub const ParseTask = struct {
                 }, Logger.Loc{ .start = 0 });
                 return JSAst.init((try js_parser.newLazyExportAST(allocator, bundler.options.define, opts, log, root, &source, "")).?);
             },
+
+            .sqlite_embedded, .sqlite => {
+                if (!bundler.options.target.isBun()) {
+                    log.addError(
+                        null,
+                        Logger.Loc.Empty,
```

Comment:
this not having a location is lame.

<img width="595" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/730aca2b-4106-4f9d-bdb4-367ba756b753">

someone will eventually run into this error, where they want to find and remove the import.

Reply:
could add the error earlier i think